### ISSUE TITLE
Why the normalize plugin need a weight?

### ISSUE THREAD

**Issue by lcxywfe** [NVIDIA/TensorRT#431]
## Description

Why the normalize plugin need a weight of scale?  Is it duplicated with the param ```scale_filler```
```
Normalize::Normalize(const Weights* weights, int nbWeights, bool acrossSpatial, bool channelShared, float eps)
    : acrossSpatial(acrossSpatial)
    , channelShared(channelShared)
    , eps(eps)
{
    mNbWeights = nbWeights;
    ASSERT(nbWeights == 1);
    ASSERT(weights[0].count >= 1);
    mWeights = copyToDevice(weights[0].values, weights[0].count);
    cublasCreate(&mCublas);
}
```

## Environment

**TensorRT Version**:  6.0
**GPU Type**: 
**Nvidia Driver Version**: 
**CUDA Version**: 
**CUDNN Version**: 
**Operating System + Version**: 
**Python Version (if applicable)**: 
**TensorFlow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if container which image + tag)**:
